### PR TITLE
Finish documenting BakedLightmap and TextureLayered

### PR DESCRIPTION
**Pull request by clayjohn**
I still need to do BakedLightmapData, But I'm going to do that later in another PR.

### CODE REVIEW

File: doc/classes/TextureLayered.xml [hunk 1]
```diff
@@ -81,23 +88,30 @@
...
 		<member name="data" type="Dictionary" setter="_set_data" getter="_get_data" default="{&quot;depth&quot;: 0,&quot;flags&quot;: 4,&quot;format&quot;: 37,&quot;height&quot;: 0,&quot;layers&quot;: [  ],&quot;width&quot;: 0}">
+			Returns a dictionary with all the data used by this texture.
 		</member>
 		<member name="flags" type="int" setter="set_flags" getter="get_flags" default="4">
+			Specifies with [enum Flags] apply to this texture.
 		</member>
 	</members>
 	<constants>
 		<constant name="FLAG_MIPMAPS" value="1" enum="Flags">
+			Texture will generate mip-maps on creation.
```
Comment (Calinou) [inline doc/classes/TextureLayered.xml]: We seem to be writing it as "mipmap" everywhere else, so this should be changed for consistency:

```suggestion
			Texture will generate mipmaps on creation.
```

**Review by Calinou — DISMISSED** [review]

File: doc/classes/TextureLayered.xml (hunk 1 above)
Comment (clayjohn) [reply doc/classes/TextureLayered.xml]: Thanks!

File: doc/classes/TextureLayered.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 		Base class for 3D texture types.
 	</brief_description>
 	<description>
-		Base class for [Texture3D] and [TextureArray]. Cannot be used directly.
+		Base class for [Texture3D] and [TextureArray]. Cannot be used directly. But contains all the functions necessary for accessing and using [Texture3D] and [TextureArray]. Data is set on a per-layer basis. For [Texture3D]s, the layer sepcifies the depth or Z-index, they can be treated as a bunch of 2D slices. Similarly, for [TextureArray]s, the layer specifies the array layer.
```
Comment (akien-mga) [inline doc/classes/TextureLayered.xml]: I'd make this "Cannot be used directly<b>, but</b>"

File: doc/classes/TextureLayered.xml [hunk 3]
```diff
@@ -81,23 +88,30 @@
 			<argument index="1" name="layer" type="int">
 			</argument>
 			<description>
+				Sets the data for specified layer. Data takes the form of a 2-dimensional [Image] resource.
```
Comment (akien-mga) [inline doc/classes/TextureLayered.xml]: for *the* specified layer

File: doc/classes/TextureLayered.xml [hunk 4]
```diff
@@ -81,23 +88,30 @@
...
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="data" type="Dictionary" setter="_set_data" getter="_get_data" default="{&quot;depth&quot;: 0,&quot;flags&quot;: 4,&quot;format&quot;: 37,&quot;height&quot;: 0,&quot;layers&quot;: [  ],&quot;width&quot;: 0}">
+			Returns a dictionary with all the data used by this texture.
 		</member>
 		<member name="flags" type="int" setter="set_flags" getter="get_flags" default="4">
+			Specifies with [enum Flags] apply to this texture.
```
Comment (akien-mga) [inline doc/classes/TextureLayered.xml]: Specified *which*

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.1.

**Pull request merged**